core-web/libs/sdk/client/src/lib/client/sdk-js-client.ts

```diff
@@ -261,6 +235,32 @@ export class DotCmsClient {
         }
     };
 
+    constructor(
```

Comment:
Just out of curiosity... why?

Reply:
Was the lint But I will leave it as previously.

Reply:
done.